lib/core/test/integration/Test/Integration/Scenario/CLI/Wallets.hs

```diff
@@ -63,7 +64,7 @@ spec = do
             Stdout mnemonics2 <- generateMnemonicsViaCLI ["--size", size]
             let passphrase = "Secure passphrase"
 
-            (c, out, err) <- createWalletViaCLI [walletName] mnemonics mnemonics2 passphrase
+            (c, out, err) <- createWalletViaCLI [walletName] mnemonics mnemonics2 passphrase (Just "1337")
```

Comment:
This line is [too long](https://github.com/input-output-hk/cardano-wallet/wiki/Coding-Standards#limit-line-length-to-80-characters).

How about:
```hs
            (c, out, err) <- createWalletViaCLI
                [walletName] mnemonics mnemonics2 passphrase (Just "1337")
```